apps/wallet/src/modals/Swap.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FC } from 'react';
+import type { ViewStyle } from 'react-native';
+import { StyleSheet } from 'react-native';
+import { runtime } from '@walless/core';
+import type { ModalConfigs } from '@walless/gui';
+import {
+	AnimateDirections,
+	BindDirections,
+	modalActions,
+	SwipeDownGesture,
+} from '@walless/gui';
+import type { Props as SwapProps } from 'features/Swap';
+import SwapFeature from 'features/Swap';
+import { useSafeAreaInsets } from 'utils/hooks';
+
+import { ModalId } from './types';
+
+type Props = {
+	config: ModalConfigs;
+	props: SwapProps;
+};
+
+const SwapModal: FC<Props> = ({ config, props }) => {
+	const insets = useSafeAreaInsets();
+
+	const safeAreaStyle: ViewStyle = {
+		marginTop: insets.top || 20,
```

Comment:
I have seen in one of the above files, you applied the `Math.max()` to get the this number, may it use here?

Reply:
These margin/padding values are used for different purpose